packages/qwik-city/runtime/src/route-matcher.ts

```diff
@@ -144,7 +144,7 @@ function recursiveScan(
   }
   let pathIdx = pathLength;
   const sep = suffix + '/';
-  let depthWatchdog = 5;
+  let depthWatchdog = 8;
```

Comment:
I think you can set this to 50 and also add a console.error before line 160 when it gets to 0

Reply:
Do you have any idea about what the content of the error throws?

Reply:
we merged #6327 which remove the limit and adds a test. do we want a limit?

Reply:
So do I close this pr?

Reply:
I wanted to get his pr in since he was first-time contributor 😁 I think we should have a limit of 50 or some way to set a limit. I don't think any app will even reach 50

Reply:
@PatrickJS  still needs this limit？So do i continue this PR ?

Reply:
@JerryWu1234 I'll close the PR I know you're working on a few other things 🙏 thanks so much for helping us